readme.md

```diff
@@ -43,6 +43,9 @@
 - [`@ts-delight/switch-expr.macro`](https://www.npmjs.com/package/@ts-delight/switch-expr.macro): An expression-oriented fluent alternative to javascript's switch-statement that compiles away to ternary expressions
 - [`@ts-delight/async-to-generator.macro`](https://www.npmjs.com/package/@ts-delight/async-to-generator.macro): Transform async functions to generator functions
 
+### Object lifetime / memory management
+- [`using.macro`](https://github.com/Veetaha/using.macro): Dispose local handles automatically with C# - like [`using` declaration](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-8.0/using) in TypeScript/JavaScript.
```

Comment:
Do you think there would be a better way to phrase this, maybe similarly to your repo's description? I feel like this doesn't convey a good sense of what the macro does.

Reply:
`Call objects destructors on scope escape with automatic try-finally blocks`?
I suppose the sentence should start with a verb...

Reply:
@jgierer12 Well, I tried my best to rephrase (taking into account that I am not fluent in English). If you have better phrasing, please, suggest!

Reply:
Thanks, I think that works well.